src/util.js

```diff
@@ -26,3 +27,30 @@ export const getTileSource = (assetHref) => {
   const proxiedUri = getTileProxiedUri(assetHref);
   return TILE_SOURCE_TEMPLATE.replace("{ASSET_HREF}", proxiedUri);
 }
+
+export const convertCoordinatesToEpsg4326 = (stackObj) => {
+    const epsg_4326 = "+proj=longlat +datum=WGS84 +no_defs";
+    const epsg_32632 = "+proj=utm +zone=32 +datum=WGS84 +units=m +no_defs "
+    
+    if (stackObj !== undefined && stackObj['geometry'] !== undefined && stackObj.type === 'Feature' && stackObj.geometry['converted'] === undefined) {
+      const coordinates = stackObj.geometry.coordinates[0]
+      const coordinatesConverted = coordinates.map((elem) => {
+        return proj4(epsg_32632, epsg_4326, [elem[0], elem[1]])
```

Comment:
Are we hardcoding the original projection to the one in the example instead of extracting it from the geoJSON?

Reply:
I understood that our epsg projection would be always `epsg_32632` and not variable. Having multiple projections this `convertCoordinatesToEpsg4326` function initial part should be, as you mentioned, extracted from the geoJSON and defined whatever projecting is described in stac object. :+1:

Reply:
There is a conversation where @yellowcap was explaining this topic:

https://tesselo.slack.com/archives/C03DULNTYFQ/p1652169204327259?thread_ts=1652129431.335589&cid=C03DULNTYFQ

"usually looks like", but not necessarily

Reply:
I didn't interpretate as an affirmation that we have multiple coordinates projections and one of them would be `EPSG:3763`. In that thread i thought he was refering to the object format were the coordinates are defined, being possible to have projections defenition in `crs: {init: 'epsg:3763'}` or in `properties: {proj:epsg: 3763}` (default).

Reply:
So what is the final understanding?

Reply:
Be prepared to convert dinamically any projection to `epsg:4326`.